test/core.jl

```diff
@@ -3106,3 +3106,8 @@ end
 const DATE12003 = DateTime(1917,1,1)
 failure12003(dt=DATE12003) = Dates.year(dt)
 @test isa(failure12003(), Integer)
+
+# Test error checking in bitstype
+@test_throws ErrorException bitstype 0 SPJ001
```

Comment:
Mentioning your past experience is decreasingly relevant and increasingly annoying. I find it very hard to believe that every single idea or programming practice you ever had was burnt into your psyche from the very first day you started.